astoria/managers/astdiskd/disk_manager.py

```diff
@@ -24,6 +25,7 @@ class DiskManager(StateManager[DiskManagerMessage]):
     def _init(self) -> None:
         self._providers: List[DiskProvider] = [
             UdisksConnection(self, notify_coro=self.update_state),
+            StaticDiskProvider(self, notify_coro=self.update_state),
```

Comment:
I think I want to refactor these classes so we don't have to pass in a callable to the constructor. Definitely out of scope of this PR though.